frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelDocs.tsx

```diff
@@ -79,7 +79,7 @@ export const SidePanelDocs = (): JSX.Element => {
                         size="small"
                         value={activeMenuName ?? ''}
                         options={menuOptions.map(({ name, url }) => ({ label: name, value: url }))}
-                        className="shrink whitespace-nowrap overflow-hidden"
+                        className="ml-1 shrink whitespace-nowrap overflow-hidden"
```

Comment:
@adamleithp requesting your review because I added a small margin here at the top

| Before| After |
|--------|--------|
| <img width="448" height="40" alt="image" src="https://github.com/user-attachments/assets/bb47b5bb-dffb-4299-a844-0110274c0ec6" /> | <img width="444" height="39" alt="image" src="https://github.com/user-attachments/assets/be0e66c5-832e-4703-bc0f-6f2c048eb0f0" /> |